datacube/drivers/postgres/_connections.py

```diff
@@ -272,6 +271,8 @@ def handle_dynamic_token_authentication(engine: Engine,
     @event.listens_for(engine, "do_connect")
     def override_new_connection(dialect, conn_rec, cargs, cparams):
         # Handle IAM authentication
+        from time import clock_gettime, CLOCK_REALTIME
```

Comment:
```suggestion
        # Importing here because the function `clock_gettime` is not available on Windows
        # which shouldn't be a problem, because boto3 auth is mostly used on AWS.
        from time import clock_gettime, CLOCK_REALTIME
```